docs/cli/csv-parser-discount-code.md

```diff
@@ -0,0 +1,170 @@
+# CSV Parser Discount Code
+
+Convert [commercetools discount codes](http://dev.commercetools.com/http-api-projects-discountCodes.html) CSV data to JSON. See example below for CSV format, sample response and usage
+
+## Usage
+`npm install @commercetools/csv-parser-discount-code --global`
+
+### CLI
+```
+Usage: csvparserdiscountcode [options]
+Convert commercetools discount codes CSV data to JSON.
+
+Options:
+  --help, -h                 Show help text.                                       [boolean]
+  --version, -v              Show version number.                                  [boolean]
+  --input, -i                Path to CSV file.                            [default: "stdin"]
+  --output, -o               Path to output JSON file.                   [default: "stdout"]
+  --delimiter, -d            Used CSV delimiter for input and/or output file. [default: ","]
+  --multiValueDelimiter, -m  Used CSV delimiter in multiValue
+                               fields for input/output file.                  [default: ";"]
+  --continueOnProblems, -c   Flag if parsing should continue if module
+                               encounters an error.               [boolean] [default: false]
+  --logLevel, -l             Logging level: error, warn, info or verbose.  [default: "info"]
+```
+
+#### Info on flags
+- The `--input` flag specifies the path to the discount codes CSV file. If this flag is omitted, the module will attempt to read the data from the `standard input`.
+- The `--output` flag specifies where to output/save the parsed discount codes as JSON file. Several notes on this flag:
+  - If the file specified already exists, it will be overwritten.
+  - The default location for status report logging is the standard output.
+  - If no output path is specified, the generated codes will be logged to the standard output as a result, status reports will be logged to a `csv-parser-discount-code.log` file
+- The `--delimiter` flag specifies the delimiter used in the input file. Defaults to `','` if omitted.
+- The `--multiValueDelimiter` flag specifies the delimiter for multiValue cells in CSV. Note that only the `cartDiscounts` field if present should contain multiple values. Defaults to `';'` if omitted.
+- The `--continueOnProblems` flag specifies if the module should continue parsing discount codes if it encounters an error. Defaults to `false` if omitted.
```

Comment:
Let us add some more info about errors.

Something like the error will be collated and returned in the summaryReport or anyhow we plan to return the error back to them.